sdks/python/precommit/dataflow/py3/build.gradle

```diff
@@ -33,8 +33,13 @@ task preCommitIT(dependsOn: ['sdist', 'installGcpTest']) {
     def testOpts = [
         "--tests=${tests.join(',')}",
         "--nocapture",    // Print stdout instantly
+        "--processes=1",    // Number of tests running in parallel
```

Comment:
Why do we have to set this now, and didn't have to in the past?

Reply:
It's not required for the fix. The main reason to have this and timeout flag is to fail fast if pipeline hangs. Otherwise the build will hit Gradle timeout which is over 100mins.

Reply:
Ok I undesrtand timeout part, but not sure why we need to add `--processes` - is this a tox flag? What would happen in `--processes=2`? Do we ever want to do that?